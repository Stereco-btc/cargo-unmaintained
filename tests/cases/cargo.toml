url = "https://github.com/rust-lang/cargo"
rev = "0.74.0"
