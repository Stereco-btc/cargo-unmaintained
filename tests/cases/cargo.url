https://github.com/rust-lang/cargo
