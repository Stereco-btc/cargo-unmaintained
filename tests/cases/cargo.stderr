Scanning 355 packages and their dependencies (pass --verbose for more information)
